bin/build_website

```diff
@@ -22,11 +23,12 @@ rm -rf "${DEST_SITE_DIR}"
 mkdir -p "${DEST_SITE_DIR}"
 
 echo "Cleaning up generated files..."
-rm -rf $DOCS_DIR/generated/*.html
+rm -rf "$DOCS_DIR"/generated/*.html
 
 echo "Reading and generating godoc pages..."
-cat $DOCS_DIR/godoc_packages.txt | while read package; do
-  if [[ "${package}" =~ ^\#.* ]] || [[ "${package}" == "" ]]; then
+while read -r package "$DOCS_DIR/godoc_packages.txt"; do
+  # Note: "${package:?}" instead of "${package}" to avoice shellcheck 2154
```

Comment:
avoice?

Reply:
that's how the cool kids spell "avoid" ldo...